### ISSUE TITLE
Hono-Website: Wrong links and image references

### ISSUE THREAD

**Issue by calohmn** [eclipse-hono/hono#3413]
The Hono Website currently contains some wrong links and image references, e.g. the top left Hono logo in the documentation.

Not found are:
```
https://www.eclipse.org/hono/css/AjaxLoader.gif
https://www.eclipse.org/hono/docsgetting-started
https://www.eclipse.org/hono/getting-started/
https://www.eclipse.org/hono/img/apple-touch-icon.png
https://www.eclipse.org/hono/img/fixed-background-1.jpg
https://www.eclipse.org/hono/img/fixed-background-2.jpg
https://www.eclipse.org/hono/img/home.jpg
https://www.eclipse.org/hono/img/photogrid.jpg
https://www.eclipse.org/hono/img/texture-bw.png
https://www.eclipse.org/images/favicon.ico
https://www.eclipse.org/images/HONO-Logo_Bild-Wort_quer-w-310x120px.svg
https://www.eclipse.org/img/grabbing.png
https://www.eclipse.org/index.xml
```

**Comment by sophokles73** [comment]
Yes, I have also noticed. I am currently trying to update the relearn Hugo theme to the latest version which seems to fix some of these issues ...

**Comment by sophokles73** [comment]
@calohmn Are you using a tool for checking the links?

**Comment by calohmn** [comment]
I was using the [W3C Link Checker](https://validator.w3.org/checklink), grep'ing the result HTML to get the above list.

**Comment by sophokles73** [comment]
closed via #3414 

I have created #3417 to remove docs for outdated versions
